packages/playwright-core/src/client/.eslintrc.js

```diff
@@ -1,7 +0,0 @@
-module.exports = {
-  extends: '../../../.eslintrc.js',
```

Comment:
I'm not sure why it was pointing to the wrong guy (should be eslintrc-with-ts-config.js)